### PR TITLE
Add docs for intermission work

### LINKED ISSUE #68: Update Mod 4 intermission work
- [ ] With the team, determine what intermission work should look like. 
- [ ] Bring intermission work documents into the new curriculum site
- [ ] Update all related docs content and instructions
- [ ] Ensure all links point to new curriculum site or practice repos

UPDATE: 

In the [Scope and Sequence doc](https://docs.google.com/spreadsheets/d/1ALC19bdgy6d-30P7JJ_u338IqLH_zXnYCB1_s3W6Vgw/edit?gid=1988683582#gid=1988683582), the following items are listed as Intermission work, but you will want to confirm with @epintozzi 
- [Exploring Legacy Code](https://mod4.turing.edu/lessons/intermission/legacy-code.html)
- [Continuous Integration](https://mod4.turing.edu/lessons/Continuous-Integration.html)
- Using Postman to Collaborate (no lesson)
- Kick off personal portfolio site expectations

### PR DESCRIPTION
**Pull request by juliet-e**
# Curriculum PR Template

### Does this PR close any issues?
This would close #68 

### Description
Updated intermission work docs, pulled over necessary lessons, added some non-deliverables, etc. 

### What questions do you have/what do you want feedback on? (optional)
1. What guidelines should we give on portfolio expectations over intermission? I just gave a timebox. What else?
2. Is there more that we want to add? Do we want to include Postman lesson for mock servers?
3. This is well below 20 hours. IMO that is great. Should we add additional requirements?

### CODE REVIEW

File: module4/intermission_work/index.md [hunk 1]
```diff
@@ -1,4 +1,42 @@
...
+This prework should take no longer than 20 hours to complete the Deliverables. Intermission is absolutely your time to rest and focus on self care. Plan accordingly with your schedules so you don't add unnecessary stress.
+
+### Demo Competition
+
+If you are planning to demo a project at Demo Comp in week 1 of Mod 4, you MUST timebox your time over intermission and in week 1 to ensure you are meeting module expectations. We recommend setting aside specific times to work with your demo team over intermission to finish your demo BEFORE week 1 begins. You are responsible for balancing your time between working on the demo project and the tasks below.
+
+## Deliverables
+
+1. [Legacy Code Experience](./legacy_code): Complete exercise. Timebox: 1-2 hours. 
+2. [Continuous Integration](./continuous_integration): Review Lesson. Timebox: 2-3 hours.
```
Comment (epintozzi) [inline module4/intermission_work/index.md:17]:
```suggestion
2. [Continuous Integration](./continuous_integration): Review Lesson and complete [Exit Ticket](https://forms.gle/zZAM9Ej2WLg5VrNY8). Timebox: 2-3 hours.
```

**Review by epintozzi — COMMENTED** [review]
Couple comments

**Comment by epintozzi** [comment]
> What guidelines should we give on portfolio expectations over intermission? I just gave a timebox. What else?
Is there more that we want to add? Do we want to include Postman lesson for mock servers?
This is well below 20 hours. IMO that is great. Should we add additional requirements?

I think the current guidelines you list are good. I don't think Mod 4 has had 20 hours of intermission work in a while. Perhaps we call out timeboxing 10 hours for networking, researching companies, etc? 

Postman yes unless it fits in Mod 3 (even as async or homework) before consultancy KO

File: module4/intermission_work/index.md [hunk 2]
```diff
@@ -1,4 +1,42 @@
...
+
+All deliverables must be submitted by Saturday at 5pm MT through [this survey](https://forms.gle/Nz6FJwQZLSHZU2yu9)
+
+## Non-Deliverable but Helpful
+
+1. Refamiliarize yourself with the basics of [reading and evaluating technical documentation](./reading_documentation)
+
+2. Watch [this video](https://www.youtube.com/watch?v=eesqK59rhGA) about the HTTP request-response cycle. 
+
+3. Explore the git rebase workflow with [this article](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) and [this one, too!](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
```
[the remark this answers is not in the document]
Comment (juliet-e) [reply module4/intermission_work/index.md:28]: Let's remove it from here!

File: module4/intermission_work/index.md [hunk 3]
```diff
@@ -1,4 +1,42 @@
...
+
+### Demo Competition
+
+If you are planning to demo a project at Demo Comp in week 1 of Mod 4, you MUST timebox your time over intermission and in week 1 to ensure you are meeting module expectations. We recommend setting aside specific times to work with your demo team over intermission to finish your demo BEFORE week 1 begins. You are responsible for balancing your time between working on the demo project and the tasks below.
+
+## Deliverables
+
+1. [Legacy Code Experience](./legacy_code): Complete exercise. Timebox: 1-2 hours. 
+2. [Continuous Integration](./continuous_integration): Review Lesson. Timebox: 2-3 hours.
+3. [Personal Portfolio](https://github.com/turingschool/portfolio-template): Work through the setup and begin customizing your site. Timebox: 5-6 hours
```
[the remark this answers is not in the document]
Comment (juliet-e) [reply module4/intermission_work/index.md]: I updated this to point to correct project page and will note in this ticket that intermission work should be updated

File: module4/intermission_work/open_source.md
Comment (epintozzi) [inline module4/intermission_work/open_source.md:1]: I really like this addition

**Review by epintozzi — APPROVED** [review]

**Pull request merged**